src/zzz_od/operation/back_to_normal_world.py

```
@@ -58,25 +59,48 @@ def check_screen_and_run(self) -> OperationRoundResult:
+        if result.is_success:
+            self.click_exit_battle = True
+            return self.round_retry(result.status, wait=1)
+        if self.click_exit_battle:  # 必须置前，因为会被通用的"取消"误判
+            result = self.round_by_find_and_click_area(self.last_screenshot, '战斗-菜单', '按钮-退出战斗-确认')
             if result.is_success:
                 return self.round_retry(result.status, wait=1)
         self.click_exit_battle = False
 
-        # 大部分画面左上角都有返回按钮
-        result = self.round_by_find_and_click_area(self.last_screenshot, '菜单', '返回')
+        # 战斗菜单-脱离卡死（大世界-勘域不慎进入战斗状态时使用）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)